hdfs-executor/src/main/java/org/apache/mesos/hdfs/executor/TaskSupervisor.java

```diff
@@ -0,0 +1,52 @@
+package org.apache.mesos.hdfs.executor;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
```

Comment:
I really like breaking this logic out, but if so we need to remove this logic from startProcess. We shouldn't have it in both places. It appears that it's trying to start the task twice.